treeherder/seta/analyze_failures.py

```diff
@@ -24,6 +25,7 @@ def __init__(self, **options):
 
     def run(self):
         fixed_by_commit_jobs = get_failures_fixed_by_commit()
+        load_preseed()
```

Comment:
the problem here is that this will run every time we run analyze_failures.py (daily in automation right now).  I would prefer if we could check if a specific state is invalid prior to doing load_preseed.  Otherwise, this seems like the right direction to go in- 

@rwood-moz do you have thoughts on what to check for here, or if there is another way you would solve this?